dist/bin/generate-secure-secrets

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Helper script to securely generate random secrets for Authelia.
+
+CONFIG_DIR=$(realpath $(dirname $0)/../config)
```

Comment:
maybe an early `set -e` to fail immediately if anything fails? Helps when users have trouble.

Reply:
Good idea, thanks.